cluster/examples/kubernetes/ceph/pool.yaml

```
@@ -22,6 +22,9 @@ spec:
     # replicasPerFailureDomain: 2
     # The name of the failure domain to place further down replicas
     # subFailureDomain: host
+    # hybridStorage:
+    #   primaryDeviceClass: "ssd"
```

strings shouldn't need quotes in yaml by default
```suggestion
    #   primaryDeviceClass: ssd
```